api.swagger.yaml

```diff
@@ -5471,6 +5471,16 @@ paths:
           description: |
             The notification does not exist.
       description: Mark a notification as read.
+  '/notifications/clear_all':
+    delete:
```

Comment:
Should be a PUT request. The notifications don't actually get deleted, just marked as read